### PR TITLE
Android 15 updates

### PR DESCRIPTION
**Pull request by theimpulson**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
+androidGradlePlugin = "8.6.0"
 protobufPlugin = "0.9.4"
 ktlint = "11.5.0"
 
 # Android SDK versions
-compileSdk = "34"
-minSdk = "34"
-targetSdk = "34"
+compileSdk = "35"
+minSdk = "35"
```
Comment (grote) [inline gradle/libs.versions.toml]: Could we leave this at `34` for now please?

**Review by grote — CHANGES_REQUESTED** [review]
Awesome, thanks a lot!

Was `contactsbackup/libs/com.android.vcard.jar` still in the same place or do we need to update the comment?

File: gradle/libs.versions.toml (hunk 1 above)
Comment (theimpulson) [reply gradle/libs.versions.toml]: Done

**Comment by theimpulson** [comment]
> Awesome, thanks a lot!
> 
> Was `contactsbackup/libs/com.android.vcard.jar` still in the same place or do we need to update the comment?

It was still at the same place but I had to run `m com.android.vcard` command manually to build it.

**Comment by grote** [comment]
ARgh: `IllegalArgumentException: failed to configure com.stevesoltys.seedvault.restore.install.DeviceInfoTest.test supported languages: Package targetSdkVersion=35 > maxSdkVersion=34`

What targetSdk does the AOSP build set? is there a way we set target to 34 for now? Or can we change target only for tests?

**Comment by theimpulson** [comment]
> ARgh: `IllegalArgumentException: failed to configure com.stevesoltys.seedvault.restore.install.DeviceInfoTest.test supported languages: Package targetSdkVersion=35 > maxSdkVersion=34`
> 
> What targetSdk does the AOSP build set? is there a way we set target to 34 for now? Or can we change target only for tests?

Done

**Review by grote — APPROVED** [review]
awesome, thanks!

**Pull request merged**